dll/cpl/sysdm/general.c

```diff
@@ -498,6 +500,39 @@ static VOID GetSystemInformation(HWND hwnd)
     }
 }
 
+static VOID GetSystemVersion(HWND hwnd)
+{
+    HWND hRosVersion;
+    SIZE_T lenStr, lenVersion;
+    WCHAR szVersion[MAX_PATH];
+    PVOID szStr;
+
+    lenVersion = MultiByteToWideChar(CP_ACP, 0, KERNEL_VERSION_RC, -1, (LPWSTR)&szVersion, _countof(szVersion));
```

Comment:
Check also https://git.reactos.org/?p=reactos.git;a=blob;f=base/shell/cmd/ver.c;h=7d4e497688db763eca57fcf0823bd6edfb794ba2;hb=HEAD#l126  that uses KERNEL_VERSION_STR without extra NULL terminators, and the _BUILD string that may be interesting for knowing which precise build you're using.

Reply:
Hmm, now I also see that there is a hardcoded LTEXT resource that seems to do that too...